src/components/RightSidebar.js

```
@@ -1,16 +1,71 @@
 import React from 'react'
 import styled from 'styled-components'
+import { useStore } from '../Store'
+
+const ParticipantCard = ({
+  className,
+  participant: { id, name, avatar, isYou }
+}) => {
+  return (
+    <div className={`${className} ${isYou ? 'you' : ''}`}>
+      <p>{name}</p>
+    </div>
+  )
+}
+
+const useRightSidebar = () => {
+  const { user, participants } = useStore()
+
+  return {
+    participantEntries: participants.map(ep => ({
+      ...ep,
+      isYou: user && ep.id === user.id
+    }))
+  }
+}
 
 const RightSidebar = () => {
-  return <SidebarContainer>Right</SidebarContainer>
+  const { participantEntries } = useRightSidebar()
+
+  return (
+    <SidebarContainer>
+      {(participantEntries || []).map(p => (
```

I rather not use this type of defaults as it may obscure deeper errors, instead let's make sure we set the correct defaults up the chain (to the store)